src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -24,16 +24,16 @@ public class ActorRuntime
         private const string TraceType = "ActorRuntime";
 
         // Map of ActorType --> ActorManager.
-        private static Dictionary<string, ActorManager> actorManagers = new Dictionary<string, ActorManager>();
+        private readonly Dictionary<string, ActorManager> actorManagers = new Dictionary<string, ActorManager>();
 
-        private ActorRuntime()
+        internal ActorRuntime()
```

Comment:
Seems like its made internal for UnitTests? Can UnitTest access using ActorRuntime.Instance? My concern is that in future someone can new up another instance of ActorRuntime in product code by mistake

Reply:
This risk is low and can be enforced by adding Warning Xml doc and through review.
Looking at the unit tests being added, could you help me understand how does accumulating state impact the tests?

Reply:
Once you register an actor type, you can't *unregister* it, so using a common `ActorRuntime` instance means that tests cannot run in complete isolation; they'll always be a risk that a previous test impacts the result of a following one.  (It also tends to prevent tests from being able to run in parallel.)

Reply:
Sure, sounds good. Add a warning in the XML doc of the internal constructor.

Reply:
Warning added.